src/Components/FileInput.js/index.module.scss

```diff
@@ -0,0 +1,39 @@
+.fileLoader {
+    display: flex;
+    flex-direction: column;
+    background-color: pink;
+}
+.fileLoaderComponent{
+    position: relative;
+    height: 300px;
+    width: 90%;
+    background-color: #eee;
+    border-radius: 10px;
+    border: solid #333;
+    text-align: center;
+    color: #333;
+    align-items: center;
+    margin: 20px;
+    >button{
```

Comment:
I think you need to write `&>button {`

Reply:
scss syntax would be like this 
.parentClass {
     color: green;
     childTag {
     any css...
     }
without & >